OctopusDSC/DSCResources/cOctopusServer/cOctopusServer.psm1

```diff
@@ -1512,6 +1631,12 @@ function Test-TargetResource {
         foreach ($key in $currentResource.Keys) {
             $currentValue = $currentResource.Item($key)
             $requestedValue = $params.Item($key)
+Write-Verbose "Testing key $key"
```

Comment:
This function should be generic - it shouldn't have custom code in it.
It looks like if you drop `SSLWebListenPrefix` and just use `WebListenPrefix`, it should drop the need for this code